lms/djangoapps/discussion_api/api.py

```diff
@@ -255,9 +255,11 @@ def get_thread_list(
         view=None,
         order_by="last_activity_at",
         order_direction="desc",
+        inline_discussion_id=None,
 ):
     """
-    Return the list of all discussion threads pertaining to the given course
+    Return the list of all discussion threads pertaining to the given course (or if inline_discussion_id is provided
```

Comment:
The PEP-8 standard for docstring length is 79 characters.  Please wrap the line.